components/dada-ir/src/token_tree.rs

```diff
@@ -16,8 +11,11 @@ salsa::entity2! {
 impl<Db: ?Sized + crate::Db> salsa::DebugWithDb<Db> for TokenTree {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>, db: &Db) -> std::fmt::Result {
         let db = db.as_dyn_ir_db();
-        let file_span: FileSpan = self.span(db).in_file(self.filename(db));
-        write!(f, "Tokens({:?})", file_span.into_debug(db))
+        let file_span = self.span(db).in_file(self.filename(db));
+        f.debug_struct("TokenTree")
+            .field("span", &file_span.into_debug(db))
+            .field("tokens", &self.tokens(db).debug(db))
+            .finish()
```

Comment:
I think I intentionally kept this debug-with minimal, to avoid spamming the output

Reply:
Maybe revert this change?

Reply:
Ah I see, I can do that. Since that will turn the debug dump back to the starting point mentioned in issue #14 I misunderstood that it would be an improvement. I'll push a commit to remove this change, and look at the other syntax nodes.